Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Build Stage
```

Comment:
Please do not use git clone (for the main repo; adding outside repos should be fine) or `ldd`.